### DISCUSSION TITLE
Air raid siren in HBM's (1.7.10)

### DISCUSSION THREAD

**Discussion by letovskiiy** [HbmMods/Hbm-s-Nuclear-Tech-GIT#1376] [General]
Hello guys! 
Again, accept my suggestions:
Look, consider this possibility so that the siren can be launched from the radar at approaching missiles or various other threats not from a red stone, without a wire, please do so so that it is possible to install a siren unit in different areas, and so that it automatically triggers when there is danger, BUT there are still moments, I tested the siren from the radar when when the red wire was connected, it only worked from the departure of the rocket and before it was shot down (detonation), it was a very short sound, I wanted the siren to sound automatically immediately when the siren unit was installed and connected, when the radar detected the target, and so that it continued to sound for about one more cycle after the downing (detonation) of a missile or other aerial target, and of course it turned off later. Please provide for this, it will be more interesting, while peaceful players will feel safer 😁😁😁

**Comment by JameH2** [comment]
Im sorry that your ability to figure out how redstone logistics and sirens led you to write this suggestion 

Again. Suggestions are not allowed here

Thank you

**Reply by ghost** [reply]
where can i put suggestions?

**Reply by Vaern** [reply]
nowhere
